src/main/java/com/trzewik/blackjack/game/Game.java

```diff
@@ -58,9 +61,9 @@ private void playersBetting(){
 
     private MoveType getUserChoice(Player player){
         if (player.getCash() >= player.getBetValue() && player.getLastMove() == null){
-            return UserInputProvider.collectProperMoveType(new ArrayList<>(Arrays.asList(MoveType.STAND, MoveType.HIT, MoveType.DOUBLEDOWN)), new ArrayList<>(Arrays.asList(MoveType.STAND.getExpectedPlayerChoice(), MoveType.STAND.getExpectedPlayerChoice().toUpperCase(), MoveType.HIT.getExpectedPlayerChoice(), MoveType.HIT.getExpectedPlayerChoice().toUpperCase(), MoveType.DOUBLEDOWN.getExpectedPlayerChoice(), MoveType.DOUBLEDOWN.getExpectedPlayerChoice().toUpperCase())), MessageProvider.askPlayerForHitStandDouble, player.getName());
+            return UserInputMatcher.collectProperMoveType(new ArrayList<>(Arrays.asList(MoveType.STAND, MoveType.HIT, MoveType.DOUBLE_DOWN)), MessageProvider.askPlayerForHitStandDouble, player.getName());
```

Comment:
to co tutaj podajesz jako new ArrayList jest odpowiednikiem MoveType.values(), nie mozemy tego tutaj uzyc?